File: Iniciante/1066.py
par, impar, pos, neg = 0, 0, 0, 0

for x in range(5):
    n = int(input())
    if n % 2 == 0:
        par += 1
    else:
        impar += 1
    if n > 0:
        pos += 1
    elif n < 0:
        neg += 1

print(str(par) + " valor(es) par(es)")
print(str(impar) + " valor(es) impar(es)")
print(str(pos) + " valor(es) positivo(s)")
print(str(neg) + " valor(es) negativo(s)")

File: Iniciante/1066.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int qt_par = 0, qt_impar = 0, qt_pos = 0, qt_neg = 0;
    int i, numero;

    for(i = 0; i < 5; i++){
        cin >> numero;
        if (numero % 2 == 0)
            qt_par++;
        else
            qt_impar++;
        if (numero > 0)
            qt_pos++;
        else if(numero < 0)
            qt_neg++;
    }

    cout << qt_par << " valor(es) par(es)\n";
    cout << qt_impar << " valor(es) impar(es)\n";
    cout << qt_pos << " valor(es) positivo(s)\n";
    cout << qt_neg << " valor(es) negativo(s)\n";

    return 0;
}